apps/user-profile/src/app/pages/settings-page/settings-authorization/settings-authentication.component.ts

```diff
@@ -18,33 +23,33 @@ export class SettingsAuthenticationComponent implements OnInit {
 
   removeDialogTitle: string;
   imgAtt: Attribute;
+  textAtt: Attribute;
   imageSrc = '';
   removeDialogDescription: string;
```

Comment:
this too

Reply:
Did you mean this one `removeDialogDescription`?